pkgs/development/python-modules/pyschemes/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+
+  # tests
+, pytestCheckHook
+
+
+}:
+
+buildPythonPackage rec {
+  pname = "pyschemes";
+  version = "ca6483d13159ba65ba6fc2f77b90421c40f2bbf2";
```

```suggestion
  version = "unstable-YYYY-MM-DD";
```
Why not use 1.0.0? https://pypi.org/project/pyschemes/#history

---

there are neither tags nor releases on Github and the date of the most recent commit doesn't fit with the version 1.0.0 date of pypi